MediaBrowser.Common/Plugins/BasePlugin.cs

```diff
@@ -140,6 +132,30 @@ protected BasePlugin(IApplicationPaths applicationPaths, IXmlSerializer xmlSeria
         {
             ApplicationPaths = applicationPaths;
             XmlSerializer = xmlSerializer;
+            if (this is IPluginAssembly assemblyPlugin)
```

Comment:
Why was this moved?

Reply:
The settings of the plugin aren't available in the constructor. See #4241